### PR TITLE
IOS-4462: Added query parameters  /v2/search/extension_rates/{facility_id} request

### PR DESCRIPTION
**Pull request by acmaciek**
**Issue Link**
https://spothero.atlassian.net/browse/IOS-4462

**Description**
Product wanted these parameters to be added. They are also not optional 

https://spothero.atlassian.net/wiki/spaces/~253418099/pages/3102146728?focusedCommentId=3106930852

### CODE REVIEW

File: Sources/SpotHeroAPI/Search/Transient/Requests/SearchGetTransientExtensionRates.swift [hunk 1]
```diff
@@ -63,12 +68,27 @@ public extension SearchGetTransientExtensionRates {
...
 
-        public init(startDate: Date, endDate: Date, isOversize: Bool, rateIDs: String, extensionHours: String) {
+        let actionID: String?
+        let fingerprint: String?
+        let searchID: String?
+        let sessionID: String?
+
+        public init(startDate: Date,
+                    endDate: Date,
+                    isOversize: Bool,
```
Comment (delannoyk) [inline Sources/SpotHeroAPI/Search/Transient/Requests/SearchGetTransientExtensionRates.swift:78]: Just noticed it but should this be optional? I don't remember how it is in other calls.

**Review by delannoyk — APPROVED** [review]

File: Sources/SpotHeroAPI/Search/Transient/Requests/SearchGetTransientExtensionRates.swift (hunk 1 above)
Comment (acmaciek) [reply Sources/SpotHeroAPI/Search/Transient/Requests/SearchGetTransientExtensionRates.swift:78]: Apparently none of these should be optional

File: Sources/SpotHeroAPI/Search/Common/Models/ExtensionQuoteContainer.swift [hunk 2]
```diff
@@ -12,5 +12,5 @@ public struct ExtensionQuoteContainer: Codable {
     public let rateID: String
```
Comment (acmaciek) [inline Sources/SpotHeroAPI/Search/Common/Models/ExtensionQuoteContainer.swift:12]: How do y'all feel about `ExtensionQuoteContainer ` name 🤔

Comment (acmaciek) [reply Sources/SpotHeroAPI/Search/Common/Models/ExtensionQuoteContainer.swift:12]: I was also thinking about **ExtensionQuoteResponse** or **TransientRateExtensionResult**

**Review by delannoyk — APPROVED** [review]

**Pull request merged**